File: app/src/main/java/com/example/schedule/POJO/OK_POJO/Labels.java
package com.example.schedule.POJO.OK_POJO;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Labels {

  @SerializedName("label")
  @Expose
  private String label;

  @SerializedName("name")
  @Expose
  private String name;

  public String getLabel() {
    return label;
  }

  public void setLabel(String label) {
    this.label = label;
  }

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }

}

File: app/src/main/java/com/example/schedule/POJO/OK_POJO/Labels.kt
package com.example.schedule.POJO.OK_POJO

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Labels {
    @SerializedName("label")
    @Expose
    var label: String? = null

    @SerializedName("name")
    @Expose
    var name: String? = null

}
